src/fides/api/models/privacy_experience.py

```diff
@@ -227,10 +228,28 @@ class PrivacyExperienceConfig(
         order_by="ExperienceTranslation.created_at",
     )
 
+    @property
+    def all_regions(self) -> List[PrivacyNoticeRegion]:
+        """All regions physically linked to the PrivacyExperienceConfig regardless of which locations
+        have been configured"""
+        db = Session.object_session(self)
+        if not db:
+            # For dry update scenarios, where PrivacyExperienceConfig is not bound to a session
+            return []
+
+        return [
+            exp.region for exp in self.experiences.order_by(PrivacyExperience.region)
+        ]
```

Comment:
Use this if you want to know all regions, regardless of location filtering. For example, we may ship with more regions out of the box than the user actually configures.